project/.travis.yml.twig

```diff
@@ -35,7 +35,6 @@ env:
     - SYMFONY_DEPRECATIONS_HELPER=max[self]=0
```

Comment:
Should we move this as well? 🤔

Reply:
Yes we should, we don't want the build to fail on Travis and not locally. env vars defined in `phpunit.xml` can be overriden from the CLI, right?

Reply:
I didn't move this one because below in this file, there are some conditionals which set it to `0`.
Regarding the CLI overriding, I need to check that. I remember [some issues](https://github.com/symfony/symfony/issues?q=is%3Aissue+SYMFONY_DEPRECATIONS_HELPER) about this behavior but I haven't them currently in mind.

Reply:
I've checked this locally, seems to be working fine.
I've also removed `SymfonyTestsListener` from `phpunit.xml.dist` since this is automatically registered when using the bridge.